index.js

```diff
@@ -104,6 +106,28 @@ async function underPressure (fastify, opts) {
     })
   }
 
+  if (opts.exposeStatusDetailsRoute) {
+    fastify.route({
+      ...opts.exposeStatusDetailsRoute.routeOpts || {},
+      url: opts.exposeStatusDetailsRoute.url,
+      method: 'GET',
+      schema: Object.assign({}, opts.exposeStatusDetailsRoute.routeSchemaOpts, {
```

Comment:
`opts.exposeStatusDetailsRoute.routeSchemaOpts` and the default json should be switched otherwise the last one will win always

Reply:
It's the same logic found in https://github.com/fastify/under-pressure/pull/80/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346L93. I assumed there was some reason behind not allowing to override `response`

Reply:
I get it now, you are right!